### PR TITLE
Curation and Signature subpages

### PR DESCRIPTION
**Pull request by apolignano**
This PR removes the Curation section and moves its contents as follows:
- curation information in Overview section, top table;
- HMM information in ex Signature section, now renamed HMM Profiles;

image
image

### CODE REVIEW

File: src/components/Entry/Summary/MemberDBSubtitle/index.tsx
Comment (matthiasblum) [inline src/components/Entry/Summary/MemberDBSubtitle/index.tsx:1]: A message could be added so users visiting the `curation` subpage are informed that the information has been moved to `Overview` and `Profile HMM`.
Deleted pages are defined in `src/utils/removed-pages/index.tsx`

**Review by matthiasblum — CHANGES_REQUESTED** [review]
A few minor things to fix. Thanks!

File: src/components/Entry/HmmModels/index.js [hunk 1]
```diff
@@ -77,6 +94,52 @@ const HmmModelSection = ({ logo } /*: {logo: {}} */) => {
...
+                          data-icon="&#xf019;"
+                        />{' '}
+                        Download
+                      </Link>{' '}
+                      the raw HMM for this family
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </>
```
Comment (matthiasblum) [inline src/components/Entry/HmmModels/index.js:139]: I only now realize that the HMM download link won't be displayed if the details are not available.

Would it be possible to always show the Download row (with the link, as the first row of the table, and to show the two other rows (HMM commands and gathering thresholds) if they are available?

**Review by matthiasblum — CHANGES_REQUESTED** [review]
Almost there. Just a one or two things to address. Also, the `src/subPages/Curation` directly could be deleted.

File: src/components/Entry/HmmModels/index.js (hunk 1 above)
Comment (apolignano) [reply src/components/Entry/HmmModels/index.js:139]: Sure. The download URL also needs to be changed. It seems that it's always pointing to pfam. Changing that to

```jsx
<Link href={`${config.root.API.href}/entry/${payload?.metadata?.source_database}/${payload?.metadata?.accession}?annotation=hmm`} download={`${payload?.metadata?.accession || 'download'}.hmm.gz`}>
```

**Review by matthiasblum — APPROVED** [review]
LGTM. I just pushed a commit that fixes the external link to http://www.sequenceontology.org.

Feel free to merge when tests complete. Thanks!

**Pull request merged**